CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.2.2] - 2020-03-10
+- Grafana service waits until Grafana is reachable publicly (in case `public_access.grafana`
+ is set to true) during resource creation or update.
```

Comment:
The clarification is a bit misleading, though I don't know if it's easy to write it in very clear manner. The service is accessible "if (it is not in a VPC or public_access.grafana is true) and there's no IP filter that prevents accessing it". Come to think of it, the current logic does not properly handle the IP filter case. Presumably if IP filter is anything but 0.0.0.0/0 we should not try connecting to Grafana as it's likely to fail.

Reply:
Updated 1.2.2 version description, also IP filter related business logic was added https://github.com/aiven/terraform-provider-aiven/pull/191